worlds/cv64/options.py

```diff
@@ -506,12 +506,12 @@ class WindowColorA(Range):
     default = 8
 
 
-class DeathLink(Choice):
-    """
-    When you die, everyone dies. Of course the reverse is true too.
-    Explosive: Makes received DeathLinks kill you via the Magical Nitro explosion instead of the normal death animation.
-    """
-    display_name = "DeathLink"
+class CV64DeathLink(Choice):
+    __doc__ = (DeathLink.__doc__ +
+               "\n\n    Explosive: Makes received death links kill you via the Magical Nitro explosion " +
+               "instead of the normal death animation.")
```

Comment:
You use a different format here for some reason
```suggestion
    __doc__ = (DeathLink.__doc__ + "\n\n    Explosive: Makes received death links kill you via the Magical Nitro " +
               "explosion instead of the normal death animation.")
```

Reply:
Bump